applications/matter_weather_station/README.rst

```diff
@@ -183,14 +183,15 @@ The ``build_thingy53_nrf5340_cpuapp`` parameter specifies the output directory f
 
 Generating factory data
 =======================
+To build with custom certificates, you need to use the chip-cert tool. To do it follow the first step of the :ref:`ug_matter_device_configuring_cd_generating_steps` instruction and add the chip-cert tool to PATH when built.
```

Comment:
```suggestion
=======================

To build with custom certificates, you need to use the chip-cert tool.
Follow the first step of the :ref:`ug_matter_device_configuring_cd_generating_steps` instruction and add the chip-cert tool to PATH when built.
```
Just some style edits. You also need an empty line after the header, otherwise the doc build will create a monster.